unified-prototype/settings.gradle.kts

```diff
@@ -7,6 +7,11 @@ pluginManagement {
     includeBuild("unified-plugin")
 }
 
+plugins {
```

Comment:
Why not apply jvm ecosystem?

Reply:
Just cause it wasn't used.  I'm going to follow your suggestion about creating a `jvmApplication` plugin, and I'll apply it with that change.